filebeat/config/config.go

```diff
@@ -76,6 +76,14 @@ type HarvesterConfig struct {
 	IncludeLines       []string         `config:"include_lines"`
 	MaxBytes           int              `config:"max_bytes"`
 	Multiline          *MultilineConfig `config:"multiline"`
+	JSON               *JSONConfig      `config:"json"`
+}
+
+type JSONConfig struct {
+	TextKey       string `config:"text_key"`
```

Comment:
I was also not sure about `text_key`. It's called "text" in the code, but now I realize that in the final document it uses "message", so probably `message_key` is better.